lib/cli.js

```diff
@@ -270,6 +270,20 @@ List of characters to copy, belongs to previously declared "--font". Examples:
     help: 'Pad bitmap line endings to whole bytes.'
   });
 
+  parser.add_argument('--stride', {
+    choices: [ 4, 8, 16, 32, 64 ],
+    type: positive_int,
+    default: 1,
+    help: 'Align each glyph\'s stride to the specfied number of bytes.'
+  });
+
+  parser.add_argument('--align', {
+    choices: [ 4, 8, 16, 32, 64 ],
```

Comment:
`1` should be a valid choice as well. 

```suggestion
    choices: [1, 4, 8, 16, 32, 64 ],
```